sentences/sv/light_HassTurnOff.yaml

```diff
@@ -7,10 +7,15 @@ intents:
           - "<slå_av> [<alla>] <area> <ljuskällor>"
           - "<slå_av> <area> [<alla>] <ljuskällor>"
           - "<slå_av> [<alla>] <area>[s]<ljuskällor>"
-
         slots:
           domain: light
+
       - sentences:
           - "<slå_av> <ljuskällor> <name>"
         slots:
           domain: light
+
```

Comment:
add 
```
requires_context:
 domain: light
```

to manipulate only light domain

Reply:
nice, thanks! what do you think @larsdunemark?

Reply:
Sure that should probably be added for the sentences for turning on light.